sys/net/application_layer/ng_zep/ng_zep.c

```diff
@@ -34,10 +35,7 @@
 
 #include "net/ng_zep.h"
 
-#define ENABLE_DEBUG    (0)
-#include "debug.h"
-
-#if ENABLE_DEBUG
+#if (LOG_LEVEL > LOG_LEVEL_KERNEL)
 /* For PRIu16 etc. */
 #include <inttypes.h>
```

Comment:
As we might have those line all over the place, we could move the include to log.h...